tests/integration/test_vulnerability_detector/test_feeds/test_invalid_type_url_feeds.py

```diff
@@ -179,10 +236,47 @@ def get_configuration(request):
 
 @pytest.mark.skip(reason="It will be blocked by wazuh-qa#2178, when it was solve we can enable again this test")
 def test_invalid_type_url_feeds(clean_vuln_tables, get_configuration, configure_environment, restart_modulesd):
-    """
-    Check that when importing bad feed files, vulnerability report a log parse error otherwise they are imported
-    correctly
-    """
+    '''
+    description: Check the behavior of Vulnerability Detector when importing files of several different URL types
+                 (.mp3, .jpg, .pdf ...) as custom feed. To do this, it downloads a file with a specific extension
+                 and is inserted as a URL feed in a set of providers. Finally, a correct feed is added to check if it
+                 is imported successfully.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - clean_vuln_tables:
+            type: fixture
+            brief: Clean vulnerabilities tables.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - restart_modulesd:
+            type: callable
+            brief: Restart the `wazuh-modulesd` daemon.
+
+    assertions:
+        - Check if feed metadata configuration is 'nvd'.
+        - Check if the custom feed has a compression format.
+        - Check if feed metadata configuration is not json debian, msu or /tmp/dummy.json.
+
+    input_description:
+        - List of custom feeds and list of possible invalid types of custom lists.
+
+    expected_output:
+        - 'Add error messages to this case use. Issue: https://github.com/wazuh/wazuh/issues/5210'
+        - r'(The file|File from URL) .* was successfully uncompressed into .*'
+        - r'INFO: .* The update of the .* feed finished successfully.'
+        - r'ERROR: The .* feed couldn't be parsed from .* file'
+        - r'ERROR: CVE database could not be updated.'
```

Comment:
```suggestion
        - 'Add error messages to this case use. Issue: https://github.com/wazuh/wazuh/issues/5210'
        - r'(The file|File from URL) .* was successfully uncompressed into .*'
        - r'INFO: .* The update of the .* feed finished successfully.'
        - r'ERROR: The .* feed couldn't be parsed from .* file'
        - r'ERROR: CVE database could not be updated.'
        - r'Couldn't get the Debian feed .*'
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/pull/2357/commits/11480f9f0bed096bb3a1e57d2e74ea955f51a1ff)